docs/etica/README.md

```diff
@@ -0,0 +1,52 @@
+# Macro argomenti per il capitolo Etica
+
+## Argomenti (che reputo) da trattare:
+
+### Definizione di etica ( generico )
+
+1. Cos'è l'etica?
+2. Principi etici fondamentali (autonomia, non-maleficienza, beneficienza, giustizia)
```

Comment:
```suggestion
2. Principi etici fondamentali (autonomia, non-maleficenza, beneficienza, giustizia)
```

Fonte: https://dizionario.internazionale.it/parola/maleficenza

Reply:
```suggestion
2. Principi etici fondamentali (autonomia, non-maleficienza, beneficenza, giustizia)
```

Fonte: https://www.treccani.it/magazine/lingua_italiana/domande_e_risposte/grammatica/grammatica_083.html#:~:text=La%20pi%C3%B9%20recente%20edizione%20dello,e%20a%20questa%20ci%20si%20atterr%C3%A0.